packages/examples/node/src/server.ts

```diff
@@ -7,7 +7,7 @@ import * as http from 'http';
 import * as url from 'url';
 import * as net from 'net';
 import express from 'express';
-import * as rpc from 'vscode-ws-jsonrpc';
+import * as rpc from 'vscode-ws-jsonrpc/cjs';
```

Comment:
@CGNonofr this did not work before I added the `typedVersions`. tsc/vscode complain it is not defined

Reply:
I had the same thought. This is a node application and when I switched to modules/esm then I ran into import errors beyond `vscode-ws-jsonrpc`. Before everything was commonjs and it worked. I ensure it works the same way now, too. Here no bundler is involved (vite or webpack could handle this problem)

I need to look into module only node/ts, but I just wanted to make it work again for now and provide people with options.

Reply:
btw, I use `node --loader ts-node/esm` instead of `node -r ts-node/register` in my projects, maybe it helps?

Reply:
I can give it a try